src/app/(checkin)/checkin/validates/page.client.tsx

```
@@ -0,0 +1,710 @@
+'use client'
+
+import { useEffect, useState, useRef, useCallback, forwardRef, useImperativeHandle } from 'react'
+import { useSearchParams, useRouter } from 'next/navigation'
+import { useAuth } from '@/providers/CheckIn/useAuth'
+import { toast } from '@/hooks/use-toast'
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs'
+import { AlertTriangle, History, ChevronDown, X } from 'lucide-react'
+import type { CheckinRecord, User } from '@/payload-types'
+
+const CheckinHistory = forwardRef((props: {}, ref) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)